src/game.py

```diff
@@ -2,10 +2,12 @@ class Game:
     def hello_world(self):
         return "Hello, world!"
 
```

Comment:
Should we remove this?  It was probably there for the walking skeleton, I think its fine to nuke it as part of the first story.

Reply:
Will remove!